<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Art/icon/王者奇迹/王者奇迹/yidong/57.png</normaloff>../../../../Art/icon/王者奇迹/王者奇迹/yidong/57.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>530</y>
      <width>931</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>日志区</string>
    </property>
    <widget class="QCheckBox" name="checkBox_logSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>日志筛选</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_allLogType">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DEBUG</string>
      </property>
      <property name="icon">
       <iconset theme="applications-engineering"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>INFO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WARNING</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ERROR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CRITICAL</string>
      </property>
     </item>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>911</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_logSearch">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>721</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>830</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1511</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>501</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>功能选择区</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_funcSearch">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>支持模糊匹配</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_funcSearch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>功能搜索</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_allFuncs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>241</width>
        <height>441</height>
       </rect>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_funcTips">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>200</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>双击左侧任意功能开始</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>931</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>功能操作区</string>
     </property>
     <widget class="QTabWidget" name="tabWidget_allFuncs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>911</width>
        <height>471</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>10</y>
       <width>271</width>
       <height>491</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>进度区</string>
     </property>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>380</y>
        <width>120</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>可以快速前往当前功能的工作路径</string>
      </property>
      <property name="statusTip">
       <string>可以快速前往当前功能的工作路径</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>查看产出</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="GridProgressBar" name="progressWidget" native="true">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>50</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>功能说明</string>
    </property>
    <widget class="QLabel" name="label_curFuncId">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>51</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>当前功能实例说明</string>
     </property>
     <property name="statusTip">
      <string>当前功能实例说明</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>530</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>进度总览区</string>
    </property>
    <widget class="QListWidget" name="listWidget_progress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1536</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_files">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_settting"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_developer"/>
    <addaction name="action_user"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_files"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_settting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_developer">
   <property name="text">
    <string>开发</string>
   </property>
  </action>
  <action name="action_user">
   <property name="text">
    <string>自定义</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GridProgressBar</class>
   <extends>QWidget</extends>
   <header>sample.qt.src.pyui.GridProgressBar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
